### PR TITLE
Add polyfill for IE11

### PR DESCRIPTION
**Pull request by kamataryo**
This Pull Request apply polyfill for IE11.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,5 +33,7 @@
     "webpack": "^4.19.1",
     "webpack-stream": "^5.1.1"
   },
-  "dependencies": {}
+  "dependencies": {
+    "core-js": "^2.6.5"
```
Comment (fumikito) [inline package.json]: @kamataryo We drop all `node_moduels` on deployment.
This core-js should be devDependencies, isn't it?

Comment (kamataryo) [reply package.json]: The dependencies will be bundled with webpack, so devDependnecies don't differ from dependencies in this case. Some people regard materials to bundle as devDependencies.

Comment (kamataryo) [reply package.json]: In my opinion, we should explicit `core-js` as dependencies because the code runs inside of the Cookie Tasting production code.

Comment (fumikito) [reply package.json]: @kamataryo that make sense. But I already merged this PR and moved `core-js` to devDependencies. Will be moved back in #6

**Pull request merged**